### ISSUE TITLE
Starting pipeline only on completion_trigger

### ISSUE THREAD

**Issue by pszemko-t** [awslabs/aws-deployment-framework#308]
Hi. 
A question or feature request... TBH I have no idea how to set this up currently.
I have a config for two pipelines and the second needs to be started only if the first one finishes 
In the current setup when I upload the file to 
          bucket_name: bucket.name
          object_key: key_2/package.zip
the second pipeline starts. 
But how to configure it in a way that second will run ONLY after it was triggered from first pipeline completion_trigger ?

```
pipelines:
# prerequisites pipeline
  - name: first-pipeline
    regions: eu-west-1
    default_providers: &generic_providers
      source: &generic_source
        provider: s3
        properties:
          account_id: 123412341234
          bucket_name: bucket.name
          object_key: key_1/package.zip
      build:
        provider: codebuild
        properties:
          environment_variables:
            CONTAINS_TRANSFORM: True
      deploy:
        provider: cloudformation
        properties:
          action: REPLACE_ON_FAILURE
          stack_name: adf-something-prereq
    params: 
      restart_execution_on_update: True
    targets: &generic_config_targets
      - path: /deployment
        name: Deploy-to-deployment
    completion_trigger:
      pipelines:
        - config-eu-west-1

  - name: second-pipeline
    regions: eu-west-1
    default_providers: &generic_config_providers
      source:
        provider: s3
        properties:
          account_id: 123412341234
          bucket_name: bucket.name
          object_key: key_2/package.zip
      build:
        provider: codebuild
        properties:
          environment_variables:
            CONTAINS_TRANSFORM: True
      deploy:
        provider: cloudformation
        properties:
          action: REPLACE_ON_FAILURE
          stack_name: adf-something
    params:
      restart_execution_on_update: True
    targets: *generic_config_targets
```

**Comment by sbkok** [comment]
Hi @cloudedtales,

Unfortunately this is not supported at the moment.
I had a look at the source code, it generates the Source configuration for S3 [here](https://github.com/awslabs/aws-deployment-framework/blob/31f4719f83e80bd165b1567fed27de8eff090460/src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_constructs/adf_codepipeline.py#L67-L68).
That would need to set the `PollForSourceChanges` property to `False` in order to stop CodePipeline from polling the target bucket/object.

We don't support this for the other source providers either as far as I am aware.
This would be a great first PR to add support for this, as the complexity would be low.
Ideally enabling the same support for the other source providers where that is possible too :)

Best regards, Simon

**Comment by cloudedtales** [comment]
Thx. 
It is tempting to add this :)
I'll take a look.
Regards, Przemek

**Comment by sbkok** [comment]
Thank you for your patience. I am happy to inform you that this feature has been [released as part of v3.2.0](https://github.com/awslabs/aws-deployment-framework/releases/tag/v3.2.0) just now.
I'm hereby closing this issue. Please open a new issue if you are experiencing any issues related to this feature.
